service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java

```diff
@@ -81,7 +80,7 @@ public DigitalTwinAsyncClient(@NonNull String connectionString) {
      * @param <T> The generic type to deserialize the application/json into.
      * @return A {@link ServiceResponseWithHeaders} representing deserialized application/json of the digital twin with {@link DigitalTwinGetHeaders}.
      */
-    public <T> Observable<ServiceResponseWithHeaders<T, DigitalTwinGetHeaders>> getDigitalTwinWithResponse (@NonNull String digitalTwinId, Class<T> clazz)
+    public <T> Observable<ServiceResponseWithHeaders<T, DigitalTwinGetHeaders>> getDigitalTwinWithResponse (String digitalTwinId, Class<T> clazz)
```

Comment:
argument check for clazz?

Reply:
Not in this PR. Still need to evaluate it. I was stuck with mvn clean. Will be there in a follow up PR.